### PR TITLE
Fix the "Next" button

### PR DESCRIPTION
**Pull request by IshaanPota**
## Types of changes

- [x] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Refactor (change which changes the codebase without affecting its external behavior)
- [ ] Non-breaking change (fix or feature that would causes existing functionality to work as expected)
- [ ] Breaking change (fix or feature that would cause existing functionality to __not__ work as expected)

## Purpose
To fix the next button to only work when there are more posts to see

## Approach
Every time a page is updated, check if the next page will have posts

## Checklist

- [x] My branch follows the branch naming scheme of UCLA Radio, and can merge into `master` without error.
- [ ] My code follows the code style of this project, and I have linted it to confirm this.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

## Issue

#298 

## Platform

__Operating System:__   
__Node Version:__

### CODE REVIEW

File: client/react/frontpage/components/BlogPage.js [hunk 1]
```diff
@@ -66,6 +64,13 @@ const BlogPage = React.createClass({
 
   setPageNumber(pageNum) {
     this.setState({ page_number: pageNum });
+    console.log('right here ');
```
Comment (tanzeelak) [inline client/react/frontpage/components/BlogPage.js]: Delete `console.log`s before merging

File: client/react/frontpage/components/Pagination.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ Reusable pagination
 
 const Pagination = React.createClass({
   getInitialState: function() {
```
Comment (tanzeelak) [inline client/react/frontpage/components/Pagination.js]: returns nothing, delete this function

File: client/react/frontpage/components/Pagination.js [hunk 3]
```diff
@@ -25,22 +25,18 @@ const Pagination = React.createClass({
...
-      this.props.setPageNumber(this.props.pageNumber - 1);
-      this.setState({
-        pageNumber: this.props.pageNumber - 1,
-      });
+      this.props.decreasePageNumber();
     }
   },
   nextIsValid() {
-    return this.props.pageNumber < this.props.maxPages;
+    let postsleft = this.props.postNumber - 12 * (this.props.pageNumber + 1);
```
Comment (tanzeelak) [inline client/react/frontpage/components/Pagination.js]: make `postsleft` camelcase to be consistent with the rest of the variables. camel case means to make it like `postsLeft`

**Comment by arjun-raghavan-00** [comment]
Moved to #330.

**Pull request closed without merging**